pages/linux/trizen.md

```diff
@@ -0,0 +1,27 @@
+# trizen
+
+> Arch Linux utility for building packages from the Arch User Repository (AUR).
+
+- Synchronize and update all AUR packages (excludes non-AUR packages):
```

Comment:
maybe we can update the description to `Synchronize and update all non-AUR packages` ? Then we don't need to mention "includes AUR packages" for every description after this.